### DISCUSSION TITLE
the glacier module in model

### DISCUSSION THREAD

**Discussion by Chengwenju** [iiasa/CWatM#44] [Q&A]
hello, everyone
I'm a new user for CWatM model. i have a question that whether the glacier module is considered in the model?

**Comment by sarah-hanus** [comment]
Hi,
there is very simplified glacier representation currently used in CWatM (https://github.com/iiasa/CWatM/blob/ad5a666c11f5479642042239c05e780ed2f19fc4/cwatm/hydrological_modules/snow_frost.py#L228) which basically redistributes snow/ice from higher to lower elevations but does not account for the additional water stored in glaciers.

However, we are working on a improved representation of glaciers in CWatM.

**Comment by abbylute** [comment]
Hello, just wondering if there are any updates on the status of the improved glacier module?
Thanks!
